packages/amplication-data-service-generator/src/server/static/src/auth/jwt/jwt.strategy.ts

```diff
@@ -0,0 +1,33 @@
+import { ExtractJwt, Strategy } from "passport-jwt";
+import { PassportStrategy } from "@nestjs/passport";
+import { Injectable, UnauthorizedException } from "@nestjs/common";
+import { UserInfo } from "../UserInfo";
+// @ts-ignore
```

Comment:
why do we need this?

Reply:
it a must because we generate the user service